### DISCUSSION TITLE
how to input f10 on vm with mac

### DISCUSSION THREAD

**Discussion by Francou2021** [utmapp/UTM#4709] [UTM Help (macOS)] [answered]
I utm windows 11 arm installed but need to inshift function 10 key in order to bypass wifi signin

How do I this please advance

**Comment by bzh32** [comment]
You should be able to open command prompt with ctrl alt w. If this does not work type ctrl + alt + n

**Comment by Francou2021** [comment]
I am on mac os not pc please step on how input f10 command in windows 11 so I can run oobe command

**Comment by osy** [comment, marked as answer]
Press fn+the key on the top row. If you have a touch bar, pressing fn will reveal the F keys.
